src/services/axiosInstance.ts

```diff
@@ -0,0 +1,10 @@
+import axios, { AxiosInstance } from "axios";
+const apiKey = process.env.API_KEY;
+const endPoint = `https://www.alphavantage.co/query?apikey=${apiKey}&`;
+
+export const stocksClient = (period: string, symbol: string): AxiosInstance => {
+  const axiosInstance = axios.create({
+    baseURL: `${endPoint}function=TIME_SERIES_${period}&symbol=${symbol}`,
```

Comment:
create instance only with base url not with `endpoint`
```suggestion
import axios, { AxiosInstance } from "axios";
const baseUrl = "https://www.alphavantage.co";

export const stocksClient = (): AxiosInstance => {
  const axiosInstance = axios.create({
    baseURL
  });
  return axiosInstance;
};
```